packages/ui-components/src/Sample/Sample.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from "react";
+import classNames from "classnames/bind";
```

Comment:
It might be worth noting on the main README that we need to use the [bind version of classnames](https://github.com/JedWatson/classnames#alternate-bind-version-for-css-modules) with css modules.

Reply:
Would that be to provide context for Authorship (for contributors)? I had considered introducing a CONTRIBUTING.md in this PR, but thought I might add it when actually introducing a real component. Do you think it's worth establishing those norms now?

Other things I would consider needing to go into CONTRIBUTING would be things like test setup, mentioning the option for Astroturf, what constitutes minimal documentation. What do you think?

Reply:
Adding to `CONTRIBUTING.md` would also work or even a PR template. We can wait and see if it becomes a problem though. It might not even be an issue.